.systemtest-dev.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# This is a configuration script to the system-test-driver that runs the
+# integration tests against the development version of the DC/OS UI
+#
+
+# Require a cluster to be specified from the command-line
+CLUSTER_URL="$1"
```

Comment:
Please evaluate ways to make `CLUSTER_URL` optional so that we can use the same config in combination with the cluster provisioning features that the system test driver is offering.